hudi-client/hudi-spark-client/src/main/scala/org/apache/spark/sql/hudi/SparkAdapter.scala

```diff
@@ -87,4 +88,9 @@ trait SparkAdapter extends Serializable {
    * Create Like expression.
    */
   def createLike(left: Expression, right: Expression): Expression
+
+  /**
+    * Create parquet file read for hudi.
+    */
+  def createHoodieParquetFileFormat(): ParquetFileFormat
```

Comment:
We need to call this for all table and query types not just MOR snapshot relation. So, basically at following places:
```
DefaultSource#getBaseFileOnlyView
IncrementalRelation
MergeOnReadIncrementalRelation
```

Reply:
yes，we need do that。but i think this pr is more like the prototype of id-schema,  I hope we can reach a basic agreement on this or。    full spark adaption will be another pr 。

Reply:
ok,  cow   incremental/snapshort view and  mor incremetal/optimize view will added.